xcode/Subconscious/Shared/Components/Detail/DetailMetaSheet.swift

```diff
@@ -0,0 +1,328 @@
+//
+//  DetailMetaSheet.swift
+//  Subconscious (iOS)
+//
+//  Created by Gordon Brander on 3/3/23.
+//
+
+import SwiftUI
+import ObservableStore
+
+struct DetailMetaSheet: View {
+    @Environment(\.dismiss) private var dismiss
+    var state: DetailMetaSheetModel
+    var send: (DetailMetaSheetAction) -> Void
+
+    var body: some View {
+        VStack(alignment: .leading, spacing: 0) {
+            HStack {
+                VStack(alignment: .leading, spacing: AppTheme.unit2) {
+                    HStack {
+                        if let slaslink = state.address?.toSlashlink() {
```

Comment:
nitpick: typo in "slaslink" should be "slashlink"

Reply:
Fixed